test/TestUtilities/Test.Utility/Test.Utility.csproj

```diff
@@ -74,7 +74,7 @@
   <ItemGroup Condition=" '$(IsCore)' == 'true' ">
     <PackageReference Include="System.Diagnostics.Process" Version="$(SystemPackagesVersion)" />
     <PackageReference Include="System.IO.Compression.ZipFile" Version="$(SystemPackagesVersion)" />
-    <PackageReference Include="System.Security.Cryptography.Pkcs" Version="4.5.2" />
+    <PackageReference Include="System.Security.Cryptography.Pkcs" Version="5.0.0-alpha1.19473.1" />
```

Comment:
Consider using a variable to help you move these along.

Reply:
Hey, so... I think because of making this change, the build started failing, and I don't have enough context to figure out why it's freaking out when restoring now, but heng's commit didn't have this happen. Can you take a look?